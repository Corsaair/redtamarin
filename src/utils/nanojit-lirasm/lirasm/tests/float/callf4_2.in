; This Source Code Form is subject to the terms of the Mozilla Public
; License, v. 2.0. If a copy of the MPL was not distributed with this
; file, You can obtain one at http://mozilla.org/MPL/2.0/.

f4 = immf4 1 2 4 9
g4 = immf4 -0 0 0.5 1e100
i = immi 10
j = immi -1
d = immd 9.9e9
e = immd -5.5e-5
f = immf 1.1
g = immf 5.5
res = callf4 callf4_mt cdecl f i d f4 j e g g4
retf4 res
