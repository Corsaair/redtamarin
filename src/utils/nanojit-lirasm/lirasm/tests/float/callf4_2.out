Output is: 0.0900905,0.180181,0.405407,INF
